features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -114,6 +114,21 @@
     private final ClassificationEngine classificationEngine;
 
     private final int bulkRetryCount;
+    
+    private static String eventIndexName = "netflow";
+    
+    //public static void setEventIndexName(String eventIndexName) {
+	//ElasticFlowRepository.eventIndexName = eventIndexName;
+	//}
+    
+    public String getModifiedIndex()
+    {
+    	Objects.requireNonNull(eventIndexName, "null value");
+		if(!eventIndexName.equals("netflow")) {
```

Comment:
Here we can check if the String is empty once we change the default value using:

`if(!eventIndexName.isEmpty())`